src/olympia/reviewers/forms.py

```diff
@@ -268,30 +271,40 @@ def create_option(
             )
             for report in reports
         )
-        forwarded = (
-            ((f'Reasoning: {"; ".join(notes for notes in forwarded_notes)}',),)
-            if forwarded_notes
+        forwarded_or_requeued_notes = [
+            *(move.notes for move in queue_moves),
+            *(decision.private_notes for decision in requeued_decisions),
+        ]
+        internal_notes = (
+            ((f'Reasoning: {"; ".join(forwarded_or_requeued_notes)}',),)
+            if forwarded_or_requeued_notes
             else ()
         )
         appeals = (
-            (appeal_text_obj.text, appeal_text_obj.reporter_report is not None)
+            (appeal_obj.text, appeal_obj.reporter_report is not None)
             for appealed_decision in obj.appealed_decisions.all()
-            for appeal_text_obj in appealed_decision.appeals.all()
+            for appeal_obj in appealed_decision.appeals.all()
         )
         subtexts_gen = [
-            *forwarded,
+            *internal_notes,
             *(
                 (f'{"Reporter" if is_reporter else "Developer"} Appeal: {text}',)
                 for text, is_reporter in appeals
             ),
         ]
 
         label = format_html(
-            '{}{}{}<br/><span>{}</span>'
+            '(Created on {}) {}{}{}{}<br/><span>{}</span>'
             '<details><summary>Show detail on {} reports</summary><ul>{}</ul>'
             '</details>',
+            format_datetime(obj.created),
```

Comment:
I would have stopped at just the date, but datetime is what Andreas specified 🤷‍♂️